server.js

```diff
@@ -21,9 +14,47 @@ const app = express()
 app.use(cors())
 app.use(express.json())
 
-// Start defining your routes here
+// First endpoint 
+// The app.get method takes two arguments - the path and a call back function, which can be used by the frontend.
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('This is an API with data about books. All endpoints are listed here: https://express-deployment-kim.herokuapp.com/endpoints')
+})
+
+// A list of all endpoints
+app.get('/endpoints', (req, res) => {
+  res.send(listEndpoints(app))
+})
+
+// Endpoint for all books, with all data (from json file)
+app.get('/books', (req, res) => {
+  res.json(booksData)
+})
+
+// A specific book, using params
+app.get('/books/:id', (req, res) => {
+  const { id } = req.params
+
+  const book = booksData.find(item => item.bookID === +id)
+
+  if (!book) {
+    res.status(404).send('No book found with this ID')
+  } else {
+    res.json(book)
+  }
+})
+
+// Endpoint for all authors, in alpabetical order
+app.get('/books/authors/:authors/', (req, res) => {
```

Comment:
Do you need the colon here or would it also be possible to give it another name, like '/books/authors/alphabetical/'?